akka-stream/src/main/java/akka/stream/javadsl/AsPublisher.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.javadsl;
+
+public enum AsPublisher {
+  WITH_FANOUT, WITHOUT_FANOUT
+}
```

Comment:
I think we should favor idiomatic language use over our notion of consistency, i.e. enums have a clear use-case and where they don’t fit we do something else.